cosmic-applet-bluetooth/data/com.system76.CosmicAppletBluetooth.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Cosmic Applet Network
+Comment=Write a GTK + Rust application
+Type=Application
+Exec=cosmic-applet-bluetooth
+Terminal=false
+Categories=GNOME;GTK;
+Keywords=Gnome;GTK;
+# Translators: Do NOT translate or transliterate this text (this is an icon file name)!
+Icon=com.system76.CosmicAppletNetwork.svg
```

Comment:
Same here (and in the `Name` key).

(And I guess we should update the categories,keyworks,comment as well, but that is probably not only affecting this applet.)

Reply:
Yup :sweat_smile: